tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data)"
+
+  #sed -i '1a set -x' "${rootfs}/usr/sbin/apache2ctl"
```

Comment:
We should probably remove this commented out sed statement from the tests once we have everything fixed.